site/src/components/Carousel.js

```diff
@@ -0,0 +1,165 @@
+import React, { useEffect, useState, useMemo } from 'react';
+import carouselStyles from '@site/src/components/css/Carousel.module.css';
+
+export default function Carousel({itemList, title}) {
+    
+    var itemWidthVar = 380;
+    var slidePaddingVar = 25;
+    var navButtonSizeVar = 50;
+    const Svg = require('@site/static/img/amoeba.svg').default;
+    const [currentIndex, setCurrentIndex] = useState(itemList.length * 2);
+    const [hasTransitionClass, setHasTransitionClass] = useState(true);
+    const [navDisabled, setNavDisabled] = useState(false);
+    const [slidePadding, setSlidePadding] = useState(slidePaddingVar);
+    const [itemWidth, setItemWidth] = useState(itemWidthVar);
+    const [itemHeight, setItemHeight] = useState(itemWidth * (2 / 3));
+    const [carouselWidth, setCarouselWidth] = useState(70);
```

Comment:
Whats the reasoning for using static values here `70` and `2/3 width` ?

Reply:
they're fairly arbitrary. The 2/3 is the aspect ratio for the carousel item card and the 70 is the percent width of the carousel container. I need to set an initial state here so I set it to those values. It essentially just translates to css.